app/Resources/views/groupManagment/detailGroup.html.twig

```diff
@@ -1,73 +1,67 @@
 {% extends "includes/base.html.twig" %}
 
-{% block title %}group{% endblock %}
+{% block title %}group !!!static, change!!!{% endblock %}
```

Comment:
Ist das ein TODO?

Reply:
Ja, da diese Seite von dir kam, wusste ich auf Anhieb auch keinen besseren Namen. Es ist nun ausgelagert in die Sprachdatei mit:
Group.detail.title: Meine Gruppen?!
Hast du einen besseren Vorschlag?